### DISCUSSION TITLE
How to add task to my plan

### DISCUSSION THREAD

**Discussion by Divwork92** [madrecha/jamku#1264] [QA] [answered]
How to add task to my plan

**Comment by Kishub** [comment, marked as answer]
At task list click on task name next to doggle switch and that task will be added to task plan for today 

![image](https://github.com/user-attachments/assets/e8817b86-fb4c-4356-bd47-b32177807472)

You can view the same by clicking on + icon at task bar click on work and view the same,

![image](https://github.com/user-attachments/assets/2b240cd0-91f4-4fc3-92d5-54e91fc1ce64)
